cargo-pgrx/src/command/schema.rs

```diff
@@ -458,22 +527,112 @@ fn compute_sql(
         std::process::exit(1)
     }
 
-    Ok(String::from_utf8(cargo_output.stdout).unwrap())
+    Ok(())
 }
 
-fn needs_rustc_bootstrap() -> bool {
-    if std::env::var_os("RUSTC_BOOTSTRAP").is_some_and(|var| !var.is_empty()) {
-        return false;
+fn compute_sql(profile: &CargoProfile) -> eyre::Result<()> {
+    let mut bin = get_target_dir()?;
+    bin.push(profile.target_subdir());
+    bin.push("pgrx_embed");
+
+    let mut command = std::process::Command::new(bin);
+    command.stdin(Stdio::inherit());
+    command.stdout(Stdio::inherit());
+    command.stderr(Stdio::inherit());
+
+    let command_str = format!("{:?}", command);
+    tracing::debug!(command = %command_str, "Running");
+    let embed_output = command
+        .output()
+        .wrap_err_with(|| format!("failed to spawn pgrx_embed: {}", command_str))?;
+    tracing::trace!(status_code = %embed_output.status, command = %command_str, "Finished");
+
+    if !embed_output.status.success() {
+        // We explicitly do not want to return a spantraced error here.
+        std::process::exit(1)
```

Comment:
I wish I remembered why. Let me look at the blame...

Reply:
...not sure why. It seems to be about avoiding emitting an error twice.
```suggestion
    if !embed_output.status.success() {
        // We do not want to return a spantraced error here, to
        // (speculative:) reduce the likelihood of emitting errors twice
        std::process::exit(1)
```